projects/lookup/src/modules/lookup/lookup-item-selected.pipe.ts

```diff
@@ -0,0 +1,10 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'skyLookupItemSelected',
+})
+export class SkyLookupItemSelectedPipe implements PipeTransform {
+  public transform(id: unknown, selectedIdMap: Map<unknown, unknown>): boolean {
+    return selectedIdMap.has(id);
```

Comment:
This is part of a general pattern of not calling functions in a template. Since Angular can't determine whether the result of a function would change based on the function's input, the function is called every change detection cycle, which in some cases can be quite inefficient. Using a pure pipe will ensure the `transform()` function is only called when its inputs change, thereby reducing the number of times the function is called.